### ISSUE TITLE
Project abandoned?

### ISSUE THREAD

**Issue by OtaK** [jaredhanson/locomotive#177]
Hi @jaredhanson, 

Did you abandon the project? No PR are merged, issues are accumulating and so on.

I love this framework, and maybe you could create a GitHub org and move locomotive under it, and give repo administration to a few individuals?

IMHO that would quickly solve the main issue with locomotive as it is, being upgrading to Express 4.x

Thanks,
Mathieu

**Comment by anmonteiro** [comment]
+1

**Comment by jeanfw** [comment]
+1

**Comment by rogovdm** [comment]
+1

That would be truely awesome.

**Comment by crispen-smith** [comment]
+1

This framework satisfies all my needs and gets out of the way the rest of the time.  I'd  be very happy to contribute to it in anyway that I can.

**Comment by OtaK** [comment]
@crispen-smith I feel the same about this framework. It's just Express MVC done right IMO.
I know that many have a working version (Oculus/Facebook have a working repo) based on Express 4, but I'm looking forward to bring that into the official repo's core. 

Thus the need to delegate administration rights for Locomotive to be able to go forward.
A huge concern also, is that Express 3's support might end soon, so we might get out of luck about security updates being backported.

**Comment by slyg** [comment]
Hi, perhaps it can help you to know that Viadeo has forked @jaredhanson's Locomotive: [viadeo/maglev](https://github.com/viadeo/maglev). It is actually an Express 4 compliant version of Locomotive, maintained, still kept as simple as possible, and it has been used on high-load traffic website for months.

**Comment by jaredhanson** [comment]
Yes.  I'd like to get some other contributors on this to get up-to-speed with PRs and Express 4.x support.

@slyg - Would you guys at Viadeo be willing to get your patches from Maglev submitted for merge upstream?  Who would be the primary contributors, I'll add them to the repo.

@OtaK - Do you know if Oculus/Facebook has a public repo or if they'd be willing to submit there patches upstream?

I've got a GitHub organization, [locomotivejs](https://github.com/locomotivejs), that I'll transfer the repo to and add contributors.  As always, I want to maintain the spirit of the project, primarily keeping it small and focused and not adding bloat.  We can discuss the patches as they are submitted.

The one feature I have on-going (slowly) is integration with [Electrolyte](https://github.com/jaredhanson/electrolyte) to support dependency injection into controllers, allowing easy mocking and stubbing of things such as database connections, etc.

**Comment by OtaK** [comment]
@jaredhanson Their repo is located here: https://github.com/OculusVR/locomotive. Although it's been a while they didn't commit into it, I guess their express4 code is stable. Didn't test it myself so if anyone else has feedback on their version, please give us a word.

:+1: on the maglev backports, that would be awesome

And as always, I agree on the project's spirit, it needs to stay like that, unobtrusive, but powerful for the developers.

**Comment by tregagnon** [comment]
We would be happy at Viadeo to bring back our changes from Maglev to Locomotive. We are also looking to keep it small and focused.

Here are the changes we made:
- Upgraded Express to version 4
- Allowed custom resolver for Controllers
- Fixed broken unit tests and JSHint errors
- Replaced Makefile by NPM scripts (stay in the Node.js ecosystem and work on more OS)
- Added deprecation warnings on functions mapped to deprecated Express functions

Each changes can have its own PR to allow discussions.

I think the GitHub organization is a good idea. We are not the only active fork. I hope others are willing to contribute.

@slyg and I were the ones maintaining [viadeo/maglev](https://github.com/viadeo/maglev). But @slyg recently left Viadeo for new adventures. :wink: I don't know If he plans to keep working on it. Feel free to add me as a contributor on the organization.

 @jaredhanson Do you plan to stay involved in Locomotive's development? It looks like you are busy on other projects. Maybe you don't have much time left for Locomotive?

**Comment by j3k0** [comment]
Any news on this? Too bad to see a good framework like this one being left for dead.

I was about to abandon the idea to use it until I found this thread. So if you want to keep attracting new users and not let this project die :

 * Add people as contributors (from project's settings) so you they can merge pull requests and/or push directly to this repo.
* Or forward users to one an active forks. Add a `DEPRECATED please go to <whatever active repository>` on top of the README.md.

**Comment by yangpu** [comment]
@jaredhanson This is a great project and you have done much of work. I choose to build our project on Locomotive.js since 3 years ago, however it's frustrating to see no one manage or maintain it now. 

Please let someone contribute to this project, and quick.

**Comment by cristianchaparroa** [comment]
+1

**Comment by keithics** [comment]
+1 for 2016

**Comment by rogovdm** [comment]
ping :cry:

**Comment by rogovdm** [comment]
Anybody knows about live fork of locomotive?

**Comment by crispen-smith** [comment]
I know I'm still using it but it continues to just work for me about 95% of the time.

**Comment by rogovdm** [comment]
It works for me for 100% of the time, but I want to move on and to perform dependecies updates to not to stuck.

**Comment by drudge** [comment]
I'm maintaining a fork at emvc/emvc.

**Comment by thall1961** [comment]
Is anyone still interested in moving this forward?  Seems like the first org didn't set and the forks have gone stale (is Locomotive solving an issue that's already been solved?), but I'd be happy to keep working with others on it.

**Comment by dellelce** [comment]
@jaredhanson why don't you archive this repo?

**Comment by alank64** [comment]
Not sure I understand, you need what?
